src/dragonball/src/vm/mod.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2021 Alibaba Cloud. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use serde_derive::{Deserialize, Serialize};
+
+mod kernel_config;
+pub use self::kernel_config::KernelConfigInfo;
+
+/// Configuration information for user defined NUMA nodes.
+#[derive(Clone, Debug, Default, Serialize, Deserialize, PartialEq)]
+pub struct NumaRegionInfo {
+    /// memory size for this region
+    pub size: u64,
+    /// numa node id on host for this region
+    pub host_numa_node_id: Option<u32>,
+    /// numa node id on guest for this region
+    pub guest_numa_node_id: Option<u32>,
```

Comment:
- Are these values always different?
- Can multiple `NumaRegionInfo` object contain the same values for these elements?

Reply:
- host_numa_node_id and guest_numa_node_id could be the same or different, it's related to users' actual needs.
- Technically, it won't be multiple `NumaRegionInfo` that contain the same value. We will introduce the validation check in vm configuration part PR.